app/views/playComponents/answer_section.scala.html

```diff
@@ -0,0 +1,30 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.AnswerSection
+@import views.html._
+
+@(answerSection: AnswerSection)(implicit messages: Messages)
+
+@if(answerSection.headingKey.isDefined){
+    <h2>@messages(answerSection.headingKey.get)</h2>
+}
+
+<ul class="govuk-check-your-answers cya-questions-long section">
```

Comment:
we should be using dl's for check your answers pages - https://design-system.service.gov.uk/patterns/check-answers/

If not using hmrc-frontend then I can help dig out code and classes for using dl's with assets-frontend